### ISSUE TITLE
Cosmic Text example

### ISSUE THREAD

**Issue by nicoburns** [linebender/vello#306]
It would be great to have an example of using [cosmic-text](https://github.com/pop-os/cosmic-text) with Vello. This would be super-helpful for [Blitz](https://github.com/DioxusLabs/blitz) to get high quality text layout / rendering running on top Vello Long term hopfully [parley](https://github.com/dfrg/parley) will also be an option, but it seems that at it isn't ready yet in that it is lacking both documentation and a crates.io release so for the time being cosmic-text is looking like the best option.

@dfrg I believe you had this working. Would you be willing to publish it as an example?

**Comment by dfrg** [comment]
I’d love to get this in as an example. There is a minor complication around selecting a unique id for the fonts coming from cosmic-text. This can be resolved in a few different ways— I’ll see what I can do. 

The major blocker is having simple infrastructure for spinning up new examples. Right now, we just dump everything into the with_winit crate and I’m not sure if that is the right call here.

**Comment by axelkar** [comment]
Is it possible to do layout with skrifa or rendering with COSMIC Text?

**Comment by axelkar** [comment]
Nevermind that - Do I take individual glyphs from COSMIC Text and just shove them down `vello::DrawGlyphs`?

**Comment by dfrg** [comment]
Yes, the draw glyphs API is the intended entry point for this and my prior experiment had this working with cosmic-text though I’m sure the code is massively out of date at this point.
